sources/auth.c

```diff
@@ -75,7 +76,7 @@ od_auth_frontend_cleartext(od_client_t *client)
 	kiwi_password_t client_password;
 	kiwi_password_init(&client_password);
 
-	if (client->rule->auth_query)
+    if (client->rule->auth_query)
```

Comment:
Maybe we need some checks in `od_rules_validate` to ensure we do not use auth_query and auth_pam_ together? otherwise it is a bit confusing